roles/redpanda_broker/tasks/main.yml

```diff
@@ -1,49 +1,34 @@
 ---
-- name: Check if redpanda_version is defined
-  fail:
-    msg: "A valid value must be provided for redpanda_version ex: '22.2.2-1' or 'latest'"
-  when: redpanda_version is not defined
```

Comment:
Yes, versions can still be specified. I just removed the default because I wanted to make sure users actually picked the version rather than using our default. I'll toss in another assert to ensure it is specified.